wiki/osu!_tournament_client/de.md

```diff
@@ -0,0 +1,80 @@
+# osu!-Turnier-Client
+
+Der **osu!-Turnier-Client** ist der offizielle Client, der [osu!tourney](/wiki/osu!tourney) mit zusätzlichen Kulissen und Informationen ummantelt, die für ein osu!-Turnier während eines Livestreams relevant sind.
+
+Nutzer, die Probleme mit dem Client haben, können [auf GitHub ein Issue erstellen](https://github.com/ppy/osu/issues) oder eine E-Mail an [tournaments@ppy.sh](mailto:tournaments@ppy.sh) senden.
+
+## Einstellung
+
+Um den osu!-Turnier-Client zu starten, musst du ein Startargument zur ausführbaren Datei von osu!(lazer) spezifizieren. Kreiere zu diesem Zweck eine Desktop-Verknüpfung und setze den Speicherort auf `%LOCALAPPDATA%/osulazer/osu!.exe --tournament`. Durch die spezielle Verknüpfung wird osu!(lazer) im Turnier-Client-Modus gestartet.
+
+Da der osu!-Turnier-Client nur ein Overlay für osu!tourney ist, muss das ebenfalls eingerichtet werden. Setze die Updatequelle in osu!tourney zu `Cutting Edge (Experimentell)` und erstelle eine leere Datei namens `ipc.txt` innerhalb des Installationsordners von osu!tourney. Folge danach der [osu!tourney-Einrichtungsanleitung](/wiki/osu!tourney/Setup).
+
+Öffne den osu!-Turnier-Client und du wirst diesen Konfigurationsbildschirm sehen:
+
+![Konfigurationsbildschirm des osu!-Turnier-Clients](img/setup-screen.png)
+
+- Stelle sicher, dass `Current IPC source` mit dem Speicherort der osu!tourney-Instanz übereinstimmt, die du nutzen wirst.
+- Logge dich in osu!(lazer) ein, indem du auf `Change Login` klickst.
+- Stelle den richtigen Spielmodus über das Dropdown-Menü ein.
+- Ändere die Höhe, damit sie dem Wert `Height` in der Datei `tournament.cfg` von osu!tourney entspricht.
+
+## Turniere verwalten
+
+Turnierkonfigurationen für osu!(lazer) werden in `%APPDATA%/osu/tournaments` gespeichert. Beim ersten Start des Clients wird ein Verzeichnis namens `default` innerhalb des Ordners erstellt. Ein Nutzer kann mehrere Turnierkonfigurationen verwalten und nach Bedarf zwischen ihnen wechseln, um die geeigneten Anpassungen anzuwenden.
+
+Um eine neue Turnierkonfiguration zu erstellen, lege einen Ordner mit dem Namen deines Turniers im Verzeichnis `tournaments` an.
+
+In der Turnierkonfiguration können die notwendigen Assets bereitgestellt werden, um Flaggen, Videos und Modsymbole für den Mappool anzuzeigen. Jede Asset-Kategorie hat ihren eigenen Ordner:
+
+- dein-turnier
+  - Flags
+  - Mods
+  - Videos
+
+## Personalisierung
+
+Der osu!-Turnier-Client kann personalisiert werden mit eigenen Flaggen, Modsymbolen und Videodateien. Diese werden in den entsprechenden Kulissen bei Bedarf angezeigt.
```

Comment:
```suggestion
Der osu!-Turnier-Client kann personalisiert werden mit eigenen Flaggen, Modsymbolen und Videodateien. Diese werden in den entsprechenden Szenen bei Bedarf angezeigt.
```
I'm not particularly opposed to the way you translated 'scenes' but I feel like this fit's better in this context. For example, OBS uses `Szene` here.

Reply:
I had `Szene` at first too but decided otherwise later on. Will change it back accordingly.